text/chapter10.md

```diff
@@ -57,17 +57,15 @@ We can assign this type to the function with the following foreign import declar
 {{#include ../exercises/chapter10/test/URI.purs}}
 ```
 
-We also need to write a foreign JavaScript module. If the module above is saved as `test/URI.purs`, then the foreign JavaScript module should be saved as `test/URI.js`:
+We also need to write a foreign JavaScript module to import it from. Purescript uses the CommonJS module system when interoperating with JavaScript. A corresponding foreign JavaScript module is one of the same name but extension changed from `.purs` to `.js`. If the Purescript module above is saved as `URI.purs`, then the foreign JavaScript module is saved as `URI.js`:
```

Comment:
Might be good to mention that we're planning to convert to ES modules for the next compiler release (`0.15`).
https://github.com/purescript/purescript/pull/3791

Reply:
Probably not. I don't think the "book" should account for unreleased features. When the time comes, we update the book.